frontend/src/pages/project/Settings/Danger.vue

```diff
@@ -46,11 +46,11 @@
             </div>
         </div>
 
-        <FormHeading>Export Project</FormHeading>
+        <FormHeading>Copy Project</FormHeading>
         <!-- Hiding for now (0.5) -->
         <!-- <div>
             <div>
-                <ff-button kind="secondary" @click="showExportProjectDialog()">Export Project</ff-button>
+                <ff-button kind="secondary" @click="showExportProjectDialog()">Copy Project</ff-button>
```

Comment:
```suggestion
                <ff-button kind="secondary" @click="showExportProjectDialog()">Export Project</ff-button>
```

Reverting this as Export is right for this action if it was re-enabled.